src/cluster.ts

```diff
@@ -0,0 +1,51 @@
+import { DgraphClient } from './client'
+import { Response } from "./types";
+
+/**
+ * Cluster is a wrapper over client that provides access to clusters.
+ */
+export class DgraphCluster {
+    private readonly client: DgraphClient;
+
+    constructor(client: DgraphClient) {
+        this.client = client;
+    }
+
+    /**
+     * Gets health for a single alpha instance
+     *
+     */
+    public getInstanceHealth(): Promise<Response> {
```

Comment:
see my comment in the other file.
This method and two below will become `this.client.anyClient().getHealth()` and then it's not even clear if DgraphCluster is necessary at all.